services/blockchain-indexer/methods/dataService/controllers/accounts.js

```diff
@@ -64,13 +64,29 @@ const getGenerators = async params => {
 	};
 };
 
+// eslint-disable-next-line no-unused-vars
 const getLegacyAccountInfo = async params => {
-	const legacyAccountInfo = await dataService.getLegacyAccountInfo(params);
-
-	return {
-		data: legacyAccountInfo.data,
-		meta: legacyAccountInfo.meta,
+	// const legacyAccountInfo = {
+	// 	data: {},
+	// 	meta: {},
+	// };
+
+	// const response = await dataService.getLegacyAccountInfo(params);
+	// if (response.data) legacyAccountInfo.data = response.data;
+	// if (response.meta) legacyAccountInfo.meta = response.meta;
+
+	const legacyAccountInfo = {
+		data: {
+			legacyAddress: '3057001998458191401L',
+			address: 'lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu',
+			balance: '10000000',
+		},
+		meta: {
+			publicKey: 'd5aa0d647b5d9ff0285321d606c870348711266ea8f0df627ef8f39d1c9959c7',
```

Comment:
Please move the `address` to the `response.meta`
```suggestion
			balance: '10000000',
		},
		meta: {
			address: 'lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu',
			publicKey: 'd5aa0d647b5d9ff0285321d606c870348711266ea8f0df627ef8f39d1c9959c7',
```